book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,111 +1,359 @@
-As part of the instructions, it is helpful to link to datasheets for the components/parts of your hardware and to list the tools required to assemble it.
-If the design requires specialized tools, tell people where to get them.
-- Using the hardware: Once someone has made the hardware, they need to know how to use it.
-Provide instructions that explain what it does, how to set it up, and how to interact with it.
-- Design rationale: If someone wants to modify your design, they will want to know why it is the way it is.
-Explain the overall plan of the hardware's design and why you made the specific choices you did.
-- Limit jargon: Keep in mind that these instructions may be read by someone whose expertise or training is different from yours.
-As much as possible, try to write to a general audience, check your instructions for industry jargon, and be explicit about what you assume the user knows.
-- Format: The instructions could be in a variety of formats, such as a wiki, text file, Google Doc, or PDF.
-Remember, though, that others might want to modify your instructions as they modify your hardware design, so it is good to provide the original editable files for your documentation, not just output formats like PDF.
-
-## Open source Hardware Processes and Practices
-
-### Designing your Hardware
-
-If you are planning to open source a particular piece of hardware, following certain best practices in its design will make it easier for others to make and modify the hardware:
-
-- Use free and open source software design (CAD) tools where possible: If that is not feasible, try to use low-cost and/or widely-used software packages.
-- Use standard and widely-available components, materials, and production processes: Try to avoid parts that are not available to individual customers or processes that require expensive setup costs.
-
-### Hosting Your Design Files
-
-A primary way of sharing your files is with a zip file on your website.
-While this is a great start, it makes it difficult for others to follow your progress or to contribute improvements.
-Using an online source-code repository (like GitHub, GitLab, or NotaBug) may be a better place to store your open source hardware projects.
-
-### Distributing Open Source Hardware
-
-- Provide links to the source (original design files) for your hardware on the product itself, its packaging, or its documentation.
-- Make it easy to find the source (original design files) from the website for a product.
-- Label the hardware with a version number or release date so that people can match the physical object with the corresponding version of its design files.
-- In general, clearly indicate which parts of a product are open source (and which are not).
+# Open Source Hardware
+ 
+## Definition
+
+“Open source hardware” refers to the design specifications of a physical object that are licenced such that they and the object can be studied, modified, created, and distributed by anyone. 
```

Comment:
```suggestion
“Open source hardware” (OSH) refers to the design specifications of a physical object that are licenced such that they and the object can be studied, modified, created, and distributed by anyone. 
```

Reply:
Since the acronym OSH is used in later paragraphs, good to formally define before using